Frameworks/AddressBook/_ABAddressBook.mm

```diff
@@ -0,0 +1,52 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import "_ABAddressBook.h"
+#import "UWP/WindowsApplicationModelContacts.h"
+
+@interface _ABAddressBook ()
+
+@end
+
+@implementation _ABAddressBook
```

Comment:
That makes sense then. I would prefer a more descriptive name then. Something like _ContactListManager. Having a name virtually identical to another class can create confusion going forward.

---
In reply to: [73080445](https://github.com/Microsoft/WinObjC/pull/786#discussion_r73080445) [](ancestors = 73080445,73073364)